<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>renpy解封包qt版</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>601</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>601</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: #ffffff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="will_pk_dir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>导入待封包的文件夹</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="put_rpa_dir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>选择封包导出目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cho_will_dir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cl_will_pk_dir">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cho_out_rpa_dir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cl_put_rpa_dir">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cho_rpa_out_dir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选存放目录</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="cl_ex_rpa">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>全部清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unpack_me">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>解包</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="op_unpk_out">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>打开存放目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="put_out_dir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>选择解包导出目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cho_will_pk_dir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择封包</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cl_put_out_dir">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cl_will_ex_dir">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="will_ex_dir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>导入封包</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pk_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>输入包名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cl_pk_name">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="cl_pk_rpa">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>全部清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pack_me">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>封包</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="op_rpa_out">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>打开导出目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cl_will_pk_dir</sender>
   <signal>clicked()</signal>
   <receiver>will_pk_dir</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_put_rpa_dir</sender>
   <signal>clicked()</signal>
   <receiver>put_rpa_dir</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_will_ex_dir</sender>
   <signal>clicked()</signal>
   <receiver>will_ex_dir</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_put_out_dir</sender>
   <signal>clicked()</signal>
   <receiver>put_out_dir</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_pk_rpa</sender>
   <signal>clicked()</signal>
   <receiver>will_pk_dir</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_pk_rpa</sender>
   <signal>clicked()</signal>
   <receiver>put_rpa_dir</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_ex_rpa</sender>
   <signal>clicked()</signal>
   <receiver>will_ex_dir</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_ex_rpa</sender>
   <signal>clicked()</signal>
   <receiver>put_out_dir</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_pk_name</sender>
   <signal>clicked()</signal>
   <receiver>pk_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cl_pk_rpa</sender>
   <signal>clicked()</signal>
   <receiver>pk_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
